### PR TITLE
add(epp): cisco firepower snmp

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Description

Cisco Firepower SNMP Monitoring procedure 

## Target serie

- [X] 20.04.x
- [X] 20.10.x (master)

### CODE REVIEW

File: en/integrations/plugin-packs/procedures/network-cisco-firepower-snmp.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+| Metric name            | Description                                       | Unit |
+| :--------------------- | :------------------------------------------------ | :--- |
+| fault status           | Bank status, possible to set string-based alerts  |      |
+| faults.total.count     | Number of total faults                            |      |
+| faults.info.count      | Number of informational faults                    |      |
+| faults.minor.count     | Number of minor faults                            |      |
+| faults.warning.count   | Number of warning faults                          |      |
+| faults.major.count     | Number of major faults                            |      |
+| faults.critical.count  | Number of critical faults                         |      |
+
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/network-cisco-firepower-snmp.md:35]: Extra space

File: en/integrations/plugin-packs/procedures/network-cisco-firepower-snmp.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+| :--------------------------------------- | :------------------------- | :--- |
+| *securitymodule*#memory.usage.bytes      | Memory usage               | B    |
+| *securitymodule*#memory.free.bytes       | Free memory                | B    |
+| *securitymodule*#memory.usage.percentage | Memory usage in percentage | %    |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites
+
+To control your Cisco Firepower, the SNMP must be configured.
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/network-cisco-firepower-snmp.md:81]: Any link to guide user to the official documentation. 

E.g: https://www.cisco.com/c/en/us/support/docs/ip/simple-network-management-protocol-snmp/213971-configure-snmp-on-firepower-ngfw-applian.html

File: fr/integrations/plugin-packs/procedures/network-cisco-firepower-snmp.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+| :--------------------- | :------------------------------------------------ | :--- |
+| fault status           | Bank status, possible to set string-based alerts  |      |
+| faults.total.count     | Number of total faults                            |      |
+| faults.info.count      | Number of informational faults                    |      |
+| faults.minor.count     | Number of minor faults                            |      |
+| faults.warning.count   | Number of warning faults                          |      |
+| faults.major.count     | Number of major faults                            |      |
+| faults.critical.count  | Number of critical faults                         |      |
+
+
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/network-cisco-firepower-snmp.md:36]: Extra space

File: fr/integrations/plugin-packs/procedures/network-cisco-firepower-snmp.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+| :--------------------------------------- | :------------------------- | :--- |
+| *securitymodule*#memory.usage.bytes      | Memory usage               | B    |
+| *securitymodule*#memory.free.bytes       | Free memory                | B    |
+| *securitymodule*#memory.usage.percentage | Memory usage in percentage | %    |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prérequis
+
+Afin de contrôler vos équipements Cisco Firepower, le SNMP doit être configuré.
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/network-cisco-firepower-snmp.md:81]: Add link even if in english

File: fr/integrations/plugin-packs/procedures/network-cisco-firepower-snmp.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
+---
+id: network-cisco-firepower-snmp
+title: Cisco Firepower SNMP
+---
+
+## Contenu du Plugin-Pack
+
+### Objets supervisés
+
+Cisco Firepower inclue le CPU, Hardware, Inodes, Interfaces, Memoire et Stack.
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/network-cisco-firepower-snmp.md:10]: Sounds weird nope ?

File: en/integrations/plugin-packs/procedures/network-cisco-firepower-snmp.md [hunk 6]
```diff
@@ -0,0 +1,181 @@
+---
+id: network-cisco-firepower-snmp
+title: Cisco Firepower SNMP
+---
+
+## Plugin-Pack Assets
+
+### Monitored Objects
+
+Cisco Firepower including CPU, Faults, Hardware, Interfaces, Memory.
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/network-cisco-firepower-snmp.md:10]: Maybe we can give more insights on what Firepower, FX OS is the embedded system or a function?

**Review by Sims24 — CHANGES_REQUESTED** [review]
Few things and we're good to go ! 

Thansk

File: en/integrations/plugin-packs/procedures/network-cisco-firepower-snmp.md (hunk 6 above)
Comment (garnier-quentin) [reply en/integrations/plugin-packs/procedures/network-cisco-firepower-snmp.md]: it's the OS of the firewall. but i don't think we need to mention it.

**Review by Sims24 — APPROVED** [review]

**Comment by Sims24** [comment]
Thanks!

**Pull request merged**